e2e/e2e_test.go

```diff
@@ -846,14 +867,17 @@ func Test_EVM_Wrapped_to_EVM_Token(t *testing.T) {
 	}
 
 	// Wait for validators to update DB state after Scheduled TX is mined
-	time.Sleep(30 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.